conf/portal.application.conf

```diff
@@ -151,6 +153,14 @@ kairosdb {
 kairosdb.proxy {
   filterRollups = true
   excludedTagNames = []
+
+  # For enabling/disabling usage of a particular rollup in a query
+  rollups.blacklist = [
```

Comment:
I figured this belongs in the `proxy` config rather than the `rollups` config, since really it's all used on the proxy side when querying kairos.

The labels for periods lead to a somewhat annoying inconsistency (`hours` vs `hourly` in the rollup config), but that's because I wanted to continue to use `SamplingUnit` from the proxy code instead of importing `RollupPeriod`.

Reply:
I think it is surprising that an empty array leads to suppressing all rollup usage. At the very least the documentation should make that clear. Ideally, periods should then be optional, and its absence would imply ALL, but an empty array would be a no-op.

Reply:
Perhaps there's a misunderstanding or I can make this clearer? The presence of any value in this list can only disable rollups, the absence cannot do the same.

**Example 1:** hourly rollups are disabled for `some_metric`
```
rollups.blacklist = [
      {
         pattern = "^some_metric$",
         periods = ["hours"],
      }
  ]
]
```

**Example 2:** A pattern matches `some_metric`, but the disabled periods list is empty so nothing is disabled. (This is a strange case, but legal)
```
rollups.blacklist = [
      {
         pattern = "^some_metric$",
         periods = [],
      }
  ]
]
```

**Example 3:** Empty config. *No* rollups are disabled for any metric
```
rollups.blacklist = []
```

I would consider making Example 2 invalid, since it doesn't convey any useful information and could be misleading.

Reply:
Sorry, so it's not possible to disable all roll-up periods for a metric; e.g.

```
rollups.blacklist = [
      {
         pattern = "^some_metric$",
      }
  ]
]
```